compiler/rustc_ast/src/visit.rs

```diff
@@ -15,8 +15,8 @@
 
 use crate::ast::*;
 
-use core::ops::ControlFlow;
-
+use rustc_ast_ir::visit::VisitorResult;
```

Comment:
```suggestion
pub use rustc_ast_ir::visit::VisitorResult;
```

And then get rid of all the direct dependencies on rustc_ast_ir

Reply:
Done.